### PR TITLE
Add CheckL7ILBFrontendConfig to check-gke-ingress

### PR DESCRIPTION
**Pull request by songrx1997**
/assign @swetharepakula

### CODE REVIEW

File: cmd/check-gke-ingress/app/ingress/rule.go [hunk 1]
```diff
@@ -113,6 +113,18 @@ func CheckFrontendConfigExistence(namespace, name string, client feconfigclient.
 	return feConfig, report.Passed, fmt.Sprintf("FrontendConfig %s/%s found", namespace, name)
 }
 
+// CheckL7ILBFrontendConfig checks whether an internal ingress has a
```
Comment (swetharepakula) [inline cmd/check-gke-ingress/app/ingress/rule.go:116]: Add that it will fail if an internal ingress has a frontend config.

**Review by swetharepakula — COMMENTED** [review]
one nit, otherwise LGTM

File: cmd/check-gke-ingress/app/ingress/rule.go (hunk 1 above)
Comment (songrx1997) [reply cmd/check-gke-ingress/app/ingress/rule.go:116]: Updated

File: cmd/check-gke-ingress/app/ingress/rule.go [hunk 2]
```diff
@@ -123,3 +135,25 @@ func getBackendConfigAnnotation(svc *corev1.Service) (string, bool) {
 	}
 	return "", false
 }
+
+// isL7ILB wether an ingress is for internal load balancing.
```
Comment (swetharepakula) [inline cmd/check-gke-ingress/app/ingress/rule.go:139]: nit: whether

**Review by swetharepakula — COMMENTED** [review]
one more nit. Also looks like there is a conflict, this might need to be rebased.

File: cmd/check-gke-ingress/app/ingress/rule.go (hunk 2 above)
Comment (songrx1997) [reply cmd/check-gke-ingress/app/ingress/rule.go]: updated.

**Review by swetharepakula — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**